spec/factories/assets.rb

```
@@ -1,6 +1,8 @@
 FactoryGirl.define do
   factory :asset do
-    asset_file { (Rails.root + 'db/fake_data/simple.json').to_s }
+    to_create(&:save)
+
+    public_url 'https://manolo.es/es/co/bar.png'
```

I didn't find `asset_file` to be needed. In fact, it's likely to be incorrect (json is not a valid asset type).

`public_url` is needed because validation checks the type. I think that validation before silently failed.